### ISSUE TITLE
BUG: API does not return data

### ISSUE THREAD

**Issue by banjtheman** [banjtheman/defundthepolice#44]
# What is the Task
The API currently does not return data due to using a different schema 

## Why do we want to do this
In order for the API to work

## How can I get started?
The api is curretly looking for the states table, but it doesnt exist
https://github.com/banjtheman/defundthepolice/issues/41#issuecomment-703915660

it needs to be updated to use budget_data
https://github.com/banjtheman/defundthepolice/blob/main/CreateDB.sql

## Definition of Done
Can run these commands

```
docker-compose -f docker-compose-api.yml up -d
curl localhost:9000/health
#reutrns health check

curl "localhost:9000/getdata?state=VA&county=Arlington"
#returns data
```

**Comment by trejavu** [comment]
Hi! I’d like to take this one.

**Comment by banjtheman** [comment]
fixed by #51
